packages/dev/docs/pages/blog/drag-and-drop.mdx

```diff
@@ -0,0 +1,244 @@
+
+## Interactions
+
+Drag and drop starts with a drag source, implemented using the [useDrag](https://react-spectrum.adobe.com/react-aria/useDrag.html) hook, which provides data to be dragged. Multiple items can be dragged at once, and each item can include several representations in different data formats so that they can be dropped in many compatible locations. The [useDrop](https://react-spectrum.adobe.com/react-aria/useDrop.html) hook can be used to implement a drop target, which accepts dragged items containing specific data types.
+
+<Anatomy role="img" aria-label="Drag and drop anatomy diagram, showing drag source, drag preview, and drop target." />
+
+For mouse and touch screen users, React Aria uses the native [HTML drag and drop API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) under the hood. This means items can be dragged within the browser window, between browser windows, or even outside the browser into external native applications (e.g. email programs). External items such as files or directories from the user’s device may also be dragged in.
+
+In addition to native drag and drop, we have also implemented keyboard and screen reader accessible interactions from scratch. Keyboard users can focus a draggable element and press the `Enter` key to start dragging it. This enters a drag and drop mode across the whole page, which allows the user to press `Tab` to navigate only between drop targets that accept the dragged data, while skipping over all other elements. This reduces the number of elements on the page that must be traversed to find a drop target, and removes the guess work often found with other alternatives such as copy and paste. Once a target is chosen, pressing the `Enter` key again performs the drop.
+
+<Video
+  src={dndKeyboard + '#t=0.1'}
+  preload="metadata"
+  controls
+  aria-label="Demo of drag and drop using a keyboard, showing list items being selected and dragged to corresponding drop targets."
+  style={{maxWidth: 'min(100%, 1920px)', display: 'block', margin: '40px auto'}} />
+
+For screen reader users, the interactions are similar. We’ve taken great care to include prompts and announcements to help guide the user through the process, which adapt to the device, and are localized into over 30 languages. Drag sources and drop targets include ARIA descriptions indicating that the user can press `Enter` or double tap to drag or drop, depending on their device. We also use an ARIA live region to announce when a drag starts, with instructions on how to navigate to perform a drop, and also announce when a drop is completed successfully or canceled by the user.
```

Comment:
If I read this correctly, you are saying the process adapts to the device and the prompts are localized into 30 languages? Also slight nit to make the last sentence a bit clearer
```suggestion
For screen reader users, the interactions are similar. We’ve taken great care to include prompts and announcements to help guide the user through the process, which adapts to the device, and are localized into over 30 languages. Drag sources and drop targets include ARIA descriptions indicating that the user can press `Enter` or double tap to drag or drop, depending on their device. We also use an ARIA live region to announce when a drag starts, with instructions on how to navigate and perform a drop, and to announce when a drop is completed successfully or canceled by the user.
```

Reply:
I think adapt should be singular: the prompts and announcements adapt to the device.

Reply:
ah gotcha, I was reading as though the process adapts to the device, should be fine then